setup.cfg

```diff
@@ -0,0 +1,29 @@
+[metadata]
+name = dftbplus-ase
+version = 0.1
+author = DFTB+ developers group
+author_email = info@dftbplus.org
+url = https://github.com/dftbplus/dftbplus-ase
+description = Interfacing DFTB+ with the Atomic Simulation Environment
+long_description = file: README.rst
+long_description_content_type = text/x-rst
+license = LGPL-3.0-or-later
+platform = any
+
+[options]
+include_package_data = True
+package_dir =
+    = src
+packages = dftbplus
+install_requires =
+    wheel
+    setuptools
```

Comment:
Those are not required for installing but for setup. The `setup_requires` keyword however shouldn't be declared here, because they are already requested with the buildsystem in the `pyproject.toml`.